### PR TITLE
add rabbitmq to list of brokers

### PR DESCRIPTION
**Pull request by vaikas**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- Add pointers to RabbitMQ broker to install and available broker.
-
-

### CODE REVIEW

File: docs/eventing/broker/README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ The GCP broker is optimized for running in GCP. For more details, refer to the [
 
 For information about the Apache Kafka broker, see [link](./kafka-broker).
 
+#### RabbitMQ broker
+
+The RabbitMQ Broker uses [RabbitMQ](https://www.rabbitmq.com/) for underlying implementation. For more details, refer to the [documentation](https://github.com/knative-sandbox/eventing-rabbitmq).
```
Comment (snneji) [inline docs/eventing/broker/README.md:54]:
```suggestion
The RabbitMQ Broker uses [RabbitMQ](https://www.rabbitmq.com/) for its underlying implementation. For more information, see the [RabbitMQ broker](https://github.com/knative-sandbox/eventing-rabbitmq) in GitHub.
```

File: docs/admin/install/install-eventing-with-yaml.md [hunk 2]
```diff
@@ -206,6 +206,11 @@ Follow the procedure for the broker of your choice:
     !!! warning
         In order to use the KafkaChannel make sure it is installed on the cluster as discussed above.
 
+=== "RabbitMQ Broker"
+
+    You can install RabbitMQ broker by following instructions [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/main/broker).
+
+    For more information, see the [RabbitMQ broker](https://github.com/knative-sandbox/eventing-rabbitmq).
```
Comment (snneji) [inline docs/admin/install/install-eventing-with-yaml.md:213]:
```suggestion
    For more information, see the [RabbitMQ broker](https://github.com/knative-sandbox/eventing-rabbitmq) in GitHub.
```

**Review by snneji — CHANGES_REQUESTED** [review]
A few small suggestions to improve the link text. Clear links are better for accessibility.

**Comment by vaikas** [comment]
Thanks! I addressed the feeback, ptal :)

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**